### PR TITLE
removal of packages moving to bottlerocket-kernel-kit

### PR DESCRIPTION
**Pull request by jmt-lab**
**Description of changes:**
Removes packages that have now moved to kernel kit:

* grub
* kernel-5.10
* kernel-5.15
* kernel-6.1
* kmod-5.10-nvidia
* kmod-5.15-nvidia
* kmod-6.1-nvidia
* linux-firmware
* microcode
* libkcapi
* shim

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by rpkelly — APPROVED** [review]

**Review by ginglis13 — APPROVED** [review]

File: Twoliter.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 schema-version = 1
-release-version = "4.1.0"
+release-version = "5.0.0"
```
Comment (cbgbt) [inline Twoliter.toml:2]: nit: Version bumps are typically (though not always!) done in a separate commit.

This otherwise LGTM!

**Review by cbgbt — APPROVED** [review]

File: Cargo.lock [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 3
+version = 4
```
Comment (bcressey) [inline Cargo.lock:3]: Will this create issues with older Rust toolchains? When was version 4 support added?

**Review by bcressey — APPROVED** [review]

File: Cargo.lock (hunk 2 above)
Comment (jmt-lab) [reply Cargo.lock:3]: O.o have no idea, i just ran this today using cargo 
1.83.0 2024-10-29

**Review by bcressey — APPROVED** [review]
